### DISCUSSION TITLE
How did you create In-Network MRF files

### DISCUSSION THREAD

**Discussion by ChristinaJNHPRI** [CMSgov/price-transparency-guide#589] [General]
Hi All, 

Trying to get an understanding of how your in-network files were created.....

The actual fee schedules/rates?
Claims?

Thank you!
Christina

**Comment by dgolden** [comment]
In-Network section is based on schedules and contractual rates as described in this document: 
    https://www.cms.gov/files/document/transparency-coverage-8-steps-508.pdf

Here is the essence:

_Plans and issuers must disclose all applicable
rates(19 ) for covered items and services provided
by in-network providers.(20) To do this, plans and
issuers should start by gathering information
from their **provider contracts, rate sheets, or
other files regarding their applicable rates** with
in-network providers for covered items and
services_

**Reply by ChristinaJNHPRI** [reply]
Thank you! I have that information. This was more of a question on if the actual fees/rate schedules were used or claims. I have heard of instances where a claims universe was built and those claims are adjudicated every month to get the "rates" for the in network file. 
Since none of the documentation state to use claims, I wanted to know if anyone approached it this way. 

--Christina

**Reply by ChristinaJNHPRI** [reply]
@shaselton-usds  Do you have any insight on this?

Thank you!

**Reply by shaselton-usds** [reply]
@ChristinaJNHPRI The in-network file should be based on contracts which usually involve a fee-schedule. Claims are only to be used in the allowed amount files.
